man/man5/zfs-module-parameters.5

```diff
@@ -702,6 +702,24 @@ Default value: \fB5\fR.
 .sp
 .ne 2
 .na
+\fBzfs_arc_pc_frac\fR (uint)
+.ad
+.RS 12n
+n/128 fraction of pagecache to reclaim arc to
+
+This tunable allows ZFS arc to play more nicely with the kernel's LRU
+pagecache. It can guarantee that the arc size won't collapse under scanning
+pressure on the pagecache, yet still allows arc to be reclaimed down to
+zfs_arc_min if necessary. This value is specified as a fraction of pagecache
+size (as measured by NR_FILE_PAGES) where that fraction may exceed 1. This
+only operates during memory pressure/reclaim.
+.sp
+Default value: \fB0\fR (disabled).
+.RE
+
```

Comment:
@dbavatar Wouldn't it be more  intuitive to express ```zfs_arc_pc_frac``` as a percentage of the size of the pagecache, and then maybe call it ```zfs_arc_pc_pct```?  It would be more congruent with other (percentage-based) tunables and could be used as:
```
uint64_t min = ptob(global_page_state(NR_FILE_PAGES)) * 100 / zfs_arc_pc_pct;
```
And documented as "percentage of the pagecache size to which the ARC may be reduced".